contractsV5/KyberTradeLogic.sol

```diff
@@ -473,11 +469,101 @@ contract KyberTradeLogic is KyberHintParser, IKyberTradeLogic, PermissionGroups
         return (reserveArr[bestReserve.index], rates[bestReserve.index], isFeePaying);
     }
 
-    function convertReserveIdToAddresses(bytes8 reserveId)
+    function parseEthToTokenHint(bytes calldata hint)
+        external
```

Comment:
I don't follow why is this function in trade logic.
it can't be in hintHandler?
only function it need from trade logic is : convertAddressToReserveId
and once trade logic implements this function, it can use it and parse hint in hint handler

or maybe I am missing something.
lets discuss.

can only function that should be in trade logic

Reply:
Ah yes. Was thinking that the external functions needed to be in KyberTradeLogic based on our last discussion. Anyway it is in KyberHintHandler now.